addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class ImLivechatChannelMemberHistory(models.Model):
+    _name = "im_livechat.channel.member.history"
+    _description = "Keep the channel member history"
+
+    channel_id = fields.Many2one("discuss.channel", ondelete="set null")
+    partner_id = fields.Many2one("res.partner", ondelete="set null")
+    guest_id = fields.Many2one("mail.guest", ondelete="set null")
+    member_id = fields.Many2one("discuss.channel.member", ondelete="set null")
+    member_type = fields.Selection([
```

Comment:
If we don't use alphabetical order, if feels these 2 should be the first 2 listed fields.